src/solver/optimisation/opt_gestion_des_bornes_cas_lineaire.cpp

```diff
@@ -226,13 +271,20 @@ void OPT_InitialiserLesBornesDesVariablesDuProblemeLineaire(PROBLEME_HEBDO* Prob
 
                 Var = CorrespondanceVarNativesVarOptim
                         ->NumeroDeVariableCoutExtremiteVersOrigineDeLInterconnexion[Interco];
-                if (CoutDeTransport->IntercoGereeAvecLoopFlow == OUI_ANTARES)
-                    Xmax[Var] = ValeursDeNTC->ValeurDeNTCExtremiteVersOrigine[Interco]
-                                + ValeursDeNTC->ValeurDeLoopFlowOrigineVersExtremite[Interco];
+
+                if (SetToZeroLinkNTCForAdequacyPatchFirstStep == Data::setToZero
+                    || SetToZeroLinkNTCForAdequacyPatchFirstStep == Data::setEndStartToZero)
+                    Xmax[Var] = 0.;
                 else
-                    Xmax[Var] = ValeursDeNTC->ValeurDeNTCExtremiteVersOrigine[Interco];
+                {
+                    if (CoutDeTransport->IntercoGereeAvecLoopFlow == OUI_ANTARES)
+                        Xmax[Var] = ValeursDeNTC->ValeurDeNTCExtremiteVersOrigine[Interco]
+                                    + ValeursDeNTC->ValeurDeLoopFlowOrigineVersExtremite[Interco];
+                    else
+                        Xmax[Var] = ValeursDeNTC->ValeurDeNTCExtremiteVersOrigine[Interco];
 
-                Xmax[Var] += 0.01;
+                    Xmax[Var] += 0.01;
+                }
```

Comment:
same here, at this point, the Var is the transportation cost

Reply:
Ejected.